src/NServiceBus.Core.Tests/Sagas/ComponentTests/When_persisting_different_sagas_with_no_defined_unique_properties.cs

```diff
@@ -0,0 +1,53 @@
+namespace NServiceBus.Persistence.ComponentTests
+{
+    using System;
+    using System.Threading.Tasks;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class When_persisting_different_sagas_with_no_defined_unique_properties : SagaPersisterTests
```

Comment:
Should we rename to reflect that what's happening is that the unique property gets defaulted? (also there can be only one unique property)

>When_persisting_different_sagas_with_defaulted_unique_propertie